documentation/polymer-templates/tutorial-component-integration.asciidoc

```diff
@@ -1,14 +1,14 @@
 ---
-title: PolymerTemplate, Detecting Component Mappings
-order: 14
+title: Detecting Component Mappings
+order: 18
```

Comment:
As this (also) applies for lit template, I think it should not be the last (or among the last) but before the polymer template specific pages, but as last of the (lit) template docs.

Reply:
@pleku good point, fixed.